test/e2e/framework/util.go

```diff
@@ -1528,13 +1528,21 @@ func podsRunning(c *client.Client, pods *api.PodList) []error {
 	// are running so non-running pods cause a timeout for this test.
 	By("ensuring each pod is running")
 	e := []error{}
+	error_chan := make(chan error)
+
 	for _, pod := range pods.Items {
-		// TODO: make waiting parallel.
-		err := WaitForPodRunningInNamespace(c, pod.Name, pod.Namespace)
+		go func() {
+			error_chan <- WaitForPodRunningInNamespace(c, pod.Name, pod.Namespace)
```

Comment:
I think `go vet` is confirming this, though the lines don't match up (possibly due to merge):

```
test/e2e/framework/util.go:1606: range variable pod captured by func literal
test/e2e/framework/util.go:1606: range variable pod captured by func literal
```